### PR TITLE
Fix nwfilter vm start case failure

### PR DESCRIPTION
**Pull request by chunfuwen**
match destination could be ipaddress or hostname by iptables -L

Signed-off-by: chunfuwen <chwen@redhat.com>

### CODE REVIEW

**Comment by chunfuwen** [comment]
Failed log:
2020-04-16 11:24:27,931 virsh            L0706 DEBUG| status: 0
2020-04-16 11:24:27,931 virsh            L0707 DEBUG| stdout: Network filter testcase undefined
2020-04-16 11:24:27,931 virsh            L0708 DEBUG| stderr: 
2020-04-16 11:24:27,935 stacktrace       L0041 ERROR| 
2020-04-16 11:24:27,935 stacktrace       L0044 ERROR| Reproduced traceback from: /var/tmp/libvirt-ci/runtest/avocado-vt/avocado-vt/avocado_vt/test.py:436
2020-04-16 11:24:27,937 stacktrace       L0047 ERROR| Traceback (most recent call last):
2020-04-16 11:24:27,937 stacktrace       L0047 ERROR|   File "/var/lib/avocado/data/avocado-vt/test-providers.d/downloads/io-github-autotest-libvirt/libvirt/tests/src/nwfilter/nwfilter_vm_start.py", line 118, in run
2020-04-16 11:24:27,937 stacktrace       L0047 ERROR|     % (expect_match, out))
2020-04-16 11:24:27,938 stacktrace       L0047 ERROR|   File "/var/tmp/libvirt-ci/runtest/avocado-vt/avocado/avocado/core/test.py", line 1030, in fail
2020-04-16 11:24:27,938 stacktrace       L0047 ERROR|     raise exceptions.TestFail(message)
2020-04-16 11:24:27,938 stacktrace       L0047 ERROR| avocado.core.exceptions.TestFail: 'ACCEPT\s*sctp\s*--\s*anywhere\s*10.0.99.3\s*DSCP\smatch\s0x03\s?sctp\sspt:35\sdpt:90\sc?t?state\sESTABLISHED\sctdir' not found in output: Chain FO-vnet0 (1 references)
2020-04-16 11:24:27,938 stacktrace       L0047 ERROR| target     prot opt source               destination         
2020-04-16 11:24:27,938 stacktrace       L0047 ERROR| ACCEPT     tcp  --  anywhere             ci-vm-10-0-99-1.hosted.upshift.rdu2.redhat.com  DSCP match 0x01 tcp dpt:kerberos ctstate ESTABLISHED ctdir REPLY
2020-04-16 11:24:27,938 stacktrace       L0047 ERROR| ACCEPT     udp  --  anywhere             ci-vm-10-0-99-2.hosted.upshift.rdu2.redhat.com  DSCP match 0x02 udp dpt:su-mit-tg ctstate ESTABLISHED ctdir REPLY
2020-04-16 11:24:27,938 stacktrace       L0047 ERROR| ACCEPT     sctp --  anywhere             ci-vm-10-0-99-3.hosted.upshift.rdu2.redhat.com  DSCP match 0x03 sctp spt:35 dpt:90 ctstate ESTABLISHED ctdir REPLY
2020-04-16 11:24:27,939 stacktrace       L0048 ERROR|

**Comment by chunfuwen** [comment]
Pass result:
DATA (filename=output.expected) => NOT FOUND (data sources: variant, test, file)
DATA (filename=stdout.expected) => NOT FOUND (data sources: variant, test, file)
DATA (filename=stderr.expected) => NOT FOUND (data sources: variant, test, file)
PASS 1-type_specific.io-github-autotest-libvirt.nwfilter_vm_start.possitive_test.new_filter.variable_notation

File: libvirt/tests/src/nwfilter/nwfilter_vm_start.py [hunk 1]
```diff
@@ -112,6 +113,23 @@ def run(test, params, env):
...
+                    hostname_info = None
+                    try:
+                        hostname_info = socket.gethostbyaddr(replace_param)
+                    except socket.error as e:
+                        logging.info("Failed to get hostname from ip address with error: %s", str(e))
+                    if hostname_info:
+                        # String is used to replace ip address
+                        replace_with = "%s|%s" % (replace_param, hostname_info[0])
+                        expect_match = expect_match.replace(replace_param, replace_with)
+                        expect_match = r"%s" % expect_match
```
Comment (dzhengfy) [inline libvirt/tests/src/nwfilter/nwfilter_vm_start.py:131]: This line can be merged with L130.

Comment (chunfuwen) [reply libvirt/tests/src/nwfilter/nwfilter_vm_start.py]: updated

**Pull request merged**